src/tests/ftest/util/soak_utils.py

```diff
@@ -1331,7 +1331,8 @@ def build_job_script(self, commands, job, nodesperjob):
             "exclude": str(self.slurm_exclude_nodes),
             "error": str(error),
             "export": "ALL",
-            "exclusive": None
+            "exclusive": None,
+            "ntasks": str(ppn)
```

Comment:
To be proper, should this actually be `nodesperjob * ppn`? I think `ntasks` is the total number of tasks. (I might be wrong)

Reply:
not clear: This is from the slurm docs; I can increase it to be np (ppn*nodesperjob)   I don't think my jobs use this for allocation of any resources but I can set it to np

SLURM_NTASKS
Set to value of the --ntasks option, if specified. Or, if either of the --ntasks-per-node or --ntasks-per-gpu options are specified, set to the number of tasks in the job.
SLURM_NTASKS_PER_CORE
Number of tasks requested per core. Only set if the --ntasks-per-core option is specified.
SLURM_NTASKS_PER_GPU
Number of tasks requested per GPU. Only set if the --ntasks-per-gpu option is specified.
SLURM_NTASKS_PER_NODE
Number of tasks requested per node. Only set if the --ntasks-per-node option is specified.

Reply:
> SLURM_NTASKS
Set to value of the --ntasks option, if specified. Or, if either of the --ntasks-per-node or --ntasks-per-gpu options are specified, set to the number of tasks in the job

I interpret that to mean `SLURM_NTASKS == total number of tasks/processes`